tests/smoke/events.js

```diff
@@ -105,6 +107,54 @@
 			editor.execCommand('nanospell');
 			// wait for the first spellcheck
 			wait();
+		},
+		'test spellcheck on element does not occur when parent block has spellcheck in progress': function() {
+			var bot = this.editorBot,
+				editor = bot.editor,
+				resumeAfter = bender.tools.resumeAfter,
+				observer = observeSpellCheckEvents(editor),
+				starterHtml = '<ol><li id="a">something<ul><li id="b">somethingnested^</li></ul></li></ol>';
+
+				bot.setHtmlWithSelection(starterHtml);
+
+				var doc = editor.document,
+				outer = doc.getById('a'),
+				inner = doc.getById('b');
+
+				resumeAfter(editor, 'spellCheckComplete', completeFirstSpellcheck);
+
+				editor.execCommand('nanospell');
+
+				// wait for initial spellcheck to complete
+				wait();
+
+				function completeFirstSpellcheck() {
+
+					observer.assert(["spellCheckComplete", "startRender", "startCheckWordsAjax", "spellCheckComplete", "startRender", "startCheckWordsAjax", "startScanWords"]);
+
+					// set outer li to show that a spellcheck is in progress
+					outer.setCustomData('spellCheckInProgress', true);
```

Comment:
Yep.  I just let the initial check complete, then reset the outer block data manually to reflect a check in progress